recipe/meta.yaml

```diff
@@ -15,11 +15,15 @@ build:
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
+  build:
+    - setuptools >=50
+    - setuptools_scm >=6
+    - wheel
```

Comment:
These packages should be in `host` not in `build`. For python packages, we use `host` when we are building them